install.sh

```diff
@@ -0,0 +1,31 @@
+#! /bin/bash
+
+# Current path
+currentPath=$(realpath ./)
+
+# Install Meteorjs
+curl https://install.meteor.com/ | sh
+export PATH=/home/bas/.meteor:$PATH
+
+# Install demeteorize
+cd ~/ && npm install demeteorizer
+
+# Go back to our project
+cd "$currentPath"
+
+# Note: When using "meteorhacks:npm"
+# to prevent the error: "unknown package: npm-container"
+# (described in https://github.com/meteorhacks/npm/issues/49)
+# either add "packages/npm-container/index.js" & "packages/npm-container/package.js" to git
+# or uncomment the two fololowing lines:
```

Comment:
Typo :)

But I don't think we should tell the user he can add these files in Git, uncomment the 2 following lines is a better approach and must be the only option IMO